src/content/events/2023-11-15_CEPLAS-ARC-Clubs.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "CEPLAS ARC Clubs"
+excerpt: "a two-day adventure to prepare your lab for the ARC universe"
+category: Training
+mode: On-site
+audience: [Users]
+when:
+    start: 2023-11-15T09:00:00
+    end: 2023-11-15T17:00:00
+location:
+  short: HHU, UoC
+  address: HHU, UoC
+#   url: https://www.ceplas.eu/en/contact/how-to-get-there
+tutors:
+  - Dominik Brilhaus (CEPLAS)
+organizer:
+  name: Dominik Brilhaus
+  affiliation: CEPLAS Data
+  url: https://www.ceplas.eu/en/research/data-science-and-data-management
+image: ~/assets/images/events/ARCClub.drawio.png
+---
+
+
+## Goals
+
+- Move existing datasets into [ARCs](./../../../implementation/AnnotatedResearchContext.html)
+- Share them via the [DataHUB](./../../../implementation/DataHub.html)
```

Comment:
Use urls based from root instead of relative

Reply:
https://github.com/nfdi4plants/nfdi4plants.github.io/pull/197/commits/d9dacdb3112ee0930f9861b024372dc0ef45073c